src/hooks/update.ts

```diff
@@ -4,5 +4,9 @@ import Plugins from '../plugins'
 
 export const update: Hook<'update'> = async function () {
   const plugins = new Plugins(this.config)
-  await plugins.update()
+  try {
+    await plugins.update()
+  } catch (e) {
+    this.error(e)
```

Comment:
```suggestion
  } catch (error) {
    this.error(error.message)
```

Reply:
The error pretty prints better & the stack trace largely useless here.

Reply:
Let me know if you object!

Reply:
Hey, sorry for the delay.
I usually prefer to let the command logger do the truncation/alteration choices, thus giving it the full error object.
It gives more flexibility to users that would have extended `Command` to have their own base `Command` class, with some custom logging things.

For example, in my company, we are wiring the error logging to our company-wide observability systems, it is valuable to have the Error object instead of just a string to extract full context.